content/post/cheers-to-small-beginnings.md

```diff
@@ -0,0 +1,55 @@
+---
+author: ricardobaeta
+date: 2018-07-09
+title: "Cheers to small beginnings!"
+image: /post/cheers-to-small-beginnings/{d}ali-banner.png
+description: "A closer look at source{d}'s rebranding."
+categories: ["design", "branding"]
```

Comment:
not 100% sure this will show up on the main category (if not we need to fix our blog, but worth checking)

Reply:
I think @dpordomingo might know?

Reply:
- Blog main page display all published posts under `content/post` directory, no matter which category they have
- Categories pages seems to not linked in our current blog (I do not remember any visible link for category listing such https://blog.sourced.tech/categories/culture/ )
- Categories were used in the landing (before the redesign), to display the latest blog articles for `culture` and `technical` categories; but now "last news" seems to be [hardcoded in the index](https://github.com/src-d/landing/blob/master/hugo/layouts/index.html#L284), 

considering the said above, categories could be ignored with our current stack

Reply:
Adding "culture", "design"